qcfractal/queue/managers.py

```diff
@@ -448,11 +497,24 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
 
             self.active -= n_result
             for key, result in results.items():
+                wall_time_seconds = 0
                 if result.success:
                     n_success += 1
+                    try:
+                        wall_time_seconds = result.provenance.wall_time
+                    except:
+                        pass
                 else:
                     error_payload.append(f"Job {key} failed: {result.error.error_type} - "
                                          f"Msg: {result.error.error_message}")
+                    try:
+                        wall_time_seconds = result.input_data['provenance'].get('wall_time')
+                    except:
+                        pass
+                try:
```

Comment:
Why a try here?

Reply:
In case for whatever reason the `wall_time` variable does not get filled in with an actual float. I'm trying to preempt getting something like a `None` here, or having a job crash because the cluster job died and something mangled came back. This is just a safety catch which won't bring down the whole manager if something goes very wrong. 

Then again, we may want it to do exactly that to better catch actual bugs. Thoughts?